packages/lit-html/src/lit-html.ts

```diff
@@ -671,8 +671,8 @@ export class NodePart {
   protected __directive?: Directive;
 
   constructor(
-    public _startNode: ChildNode,
-    public _endNode: ChildNode | null,
+    public __startNode: ChildNode,
```

Comment:
It would be great to make these private again, but define another interface that makes them public and cast to that where needed. I had some difficulty with getting TS to allow this last I tried.

Reply:
Done.